### ISSUE TITLE
BROWSER SUPPORT

### ISSUE THREAD

**Issue by COLABORATI** [cmroanirgo/inviscss#1]
Yes, you need to add information about which web browsers you are actively supporting with a css framework. Always. Nobody will use it, if you do not add that info.

**Comment by cmroanirgo** [comment]
Yes, sorry it's not clear. It's aiming for IE11+/Edge, Safari 8+ & everything modern (eg. Chrome, FF, Opera). Unfortunately, extensive IE11/Edge hasn't been done (a couple of faults are raised against them). The framework makes use of `normalize.css` which gives a common ground for most cross browser incompatibilities.

See `browserlist` in the source for the complete set of targeted platforms. 

Thanks for the feedback.
